src/UserFrontend.php

```diff
@@ -60,8 +60,20 @@ public static function login_head() {
    * @implements login_form_defaults
    */
   public static function login_form_defaults(array $args) {
+    if (file_exists(get_stylesheet_directory() . '/dist/styles/login.css')) {
+      wp_enqueue_style('theme/login', get_stylesheet_directory_uri() . '/dist/styles/login.css');
```

Comment:
This happens in line 85 already: https://github.com/netzstrategen/wordpress-core-standards/pull/9/files#diff-a4a15f99f66b9d0b641ea2f50c52c379R85